lib/main.dart

```diff
@@ -105,7 +105,11 @@ class _RandomWordsState extends State<RandomWords> {
             appBar: AppBar(
               title: const Text('Saved Suggestions'),
             ),
-            body: ListView(children: divided),
+            body: divided.isNotEmpty
+                ? ListView(children: divided)
+                : const Center(
+                    child: Text('Saved suggestions list is empty'),
```

Comment:
It would be good to add `const` before the Text constructor as its value can be determined at compile-time and optimized.